TACC/Core-Styles
# chore(elements)!: delete v0 CMS headings

## Overview / Changes

Convert headings pattern to CMS and Portal prototype.

## Related

- prototype for https://github.com/TACC/tup-ui/tree/main/apps/tup-ui headings
- prototype for https://github.com/TACC/tup-ui/tree/main/apps/tup-cms headings

## Testing

1. Open "Elements > (CMS) Headings" and "Elements > (Portal) Headings".
2. Verify headings match between the two pages.
3. Verify headings look like [Bootstrap headings](https://getbootstrap.com/docs/4.3/content/typography/#headings).
4. Verify paragraph font is smaller for CMS (12px) than Portal (whatever px, for now).

## UI

| Portal | CMS |
| - | - |
| ![portal](https://user-images.githubusercontent.com/62723358/208009591-631cc21f-394a-4701-a895-a6e374325bfd.png) | ![cms](https://user-images.githubusercontent.com/62723358/208009598-f18cbe0c-d6dd-46fe-957d-904248668b10.png) |
